server/modules/noko/projects.js

```diff
@@ -17,29 +16,37 @@ const nokoToken = process.env.NOKO_ADMIN_TOKEN;
  *
  * @returns {void}
  */
-const saveNokoProject = async (projectDetails, message, status = 'success') => ({
-  projectId: projectDetails.id,
+export const saveNokoProject = (projectDetails, message, status = 'success') => ({
+  projectId: (projectDetails && projectDetails.id) || null,
```

Comment:
Good job! Here, I think `(projectDetails && projectDetails.id)` would suffice.  If the condition fails, null is returned by default. So you can remove `|| null` at the end of line 20